### PR TITLE
Fixed setLink and setRobot issues

### PR DESCRIPTION
**Pull request by RoboDK**

### CODE REVIEW

File: Python/robodk/robolink.py [hunk 1]
```diff
@@ -5201,16 +5204,23 @@ def setRobot(self, robot=None):
...
-            if robot is None:
-                robot = Item(self.link)
-            self.link._check_connection()
-            command = 'S_Robot'
-            self.link._send_line(command)
-            self.link._send_item(self)
-            self.link._send_item(robot)
-            self.link._check_status()
+        if self.link.BUILD > 22916:
+            self.setLink(robot)
```
Comment (sambertrdk) [inline Python/robodk/robolink.py:5208]: What is the behavioural change with the old vc. new setRobot? Is it worth documenting?

File: Python/robodk/robolink.py [hunk 2]
```diff
@@ -7027,9 +7037,10 @@ def setParam(self, param, value='', skip_result=False):
                 return line
 
 
-if __name__ == "__main__":
-
-    def RoboDKInfo():
+if __name__ == "__main__":    
+    def RoboDKInfo():    
+        return
```
Comment (sambertrdk) [inline Python/robodk/robolink.py:7042]: Was this causing issues? Are there instances where we run robolink.py on its own other than to debug?

**Review by sambertrdk — APPROVED** [review]

File: Python/robodk/robolink.py (hunk 1 above)
Comment (RoboDK) [reply Python/robodk/robolink.py:5208]: They have both the same behaviour. This change is to simplify the commands RoboDK accepts in the future.

File: Python/robodk/robolink.py (hunk 2 above)
Comment (RoboDK) [reply Python/robodk/robolink.py:7042]: Running this module as an executable should not do anything. This was some leftover code by accident from manual tests we did during development.

**Pull request merged**